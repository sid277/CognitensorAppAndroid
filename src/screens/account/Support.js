import React from 'react';
import DefaultView from '../../components/default/DefaultView';
import ButtonWithIcon from '../../components/ButtonWithIcon';

const Support = () => {
  return (
    <DefaultView>
      <ButtonWithIcon title="Contact Us" iconName="phone" />
      <ButtonWithIcon title="Mail Us" iconName="mail" />
    </DefaultView>
  );
};

export default Support;
