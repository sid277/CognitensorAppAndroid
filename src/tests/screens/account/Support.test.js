import React from 'react';
import { shallow } from 'enzyme';
import Support from '../../../screens/account/Support';
import renderer from 'react-test-renderer';


test('should test Support component', () => {
const wrapper = shallow(<Support />);
expect(wrapper).toMatchSnapshot();
});