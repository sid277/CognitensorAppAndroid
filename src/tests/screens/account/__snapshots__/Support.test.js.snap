// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Support component 1`] = `ShallowWrapper {}`;
